test/polytopes.jl

```diff
@@ -264,6 +264,21 @@ end
   r = Ring(merc.([(0, 0), (1, 0), (1, 1), (0, 1)]))
   @test crs(centroid(r)) === crs(r)
 
+  # parameterization
```

Comment:
We need basic tests with `cart` coordinates. Perhaps an example with the `boundary` of the a `Box(cart(0,0), cart(1,1))` and another example with the `Rope(cart(0,0), cart(2, 0), cart(3,0))`?

Reply:
Added tests in [44c2270](https://github.com/JuliaGeometry/Meshes.jl/pull/1155/commits/44c227040cb63b1b0c33df0fc9ff57dc8c434d81).